js/app.js

```diff
@@ -0,0 +1,116 @@
+
+
+
+//person4Name
+var oReq = new XMLHttpRequest();
```

Comment:
Variable and function names should not end with a number, the names should be more descriptive.